### ISSUE TITLE
Errors with new pull request UI

### ISSUE THREAD

**Issue by dcelasun** [refined-bitbucket/refined-bitbucket#275]
The new [PR UI](https://confluence.atlassian.com/bitbucket/try-the-new-pull-request-experience-in-bitbucket-958455255.html) doesn't play well with refined-bitbucket:

```
Uncaught (in promise) TypeError: Cannot read property 'querySelector' of null
    at collapsePullRequestDescription (chrome-extension://afppminkfnfngihdocacbgeajbbdklkf/main.js:16097)
    at pullrequestRelatedFeatures (chrome-extension://afppminkfnfngihdocacbgeajbbdklkf/main.js:17485)
    at init (chrome-extension://afppminkfnfngihdocacbgeajbbdklkf/main.js:17340)
    at ./src/main.js._webextOptionsSync.default.getAll.then.options (chrome-extension://afppminkfnfngihdocacbgeajbbdklkf/main.js:17328)
```

**Comment by reyronald** [comment]
Hey @dcelasun 	, thanks for reporting!

Does it cause any issues or break anything on the new UI?

**Comment by dcelasun** [comment]
Hey, thanks for the speedy reply!

It doesn't break anything, but the extension itself doesn't work either (no syntax highlighting for example). Also, some features like diff ignore and auto collapse are builtin to Bitbucket now, so they won't be needed.

**Comment by reyronald** [comment]
@dcelasun 	To be honest I didn't know that syntax highlighting even worked in the new PR UI (haven't used it), I just assumed it didn't, but I guess they kept some of the markup intact.

I'll fix this bug, but in the meantime you can just turn off the "Description collapse" feature in the options page and it should go back to normal.

**Comment by syncrisis** [comment]
@reyronald Just a heads up, I just encountered this bug on Bitbucket Server when enabling a 3rd party plugin I've used in the past.  The new PR UI doesn't play nice with most plugins.

**Comment by rbardini** [comment]
How can I turn off the "Description collapse" feature? I can't find it in the options page, and the `options.html` file doesn't have any references to the `collapsePullRequestDescription` config.

**Comment by reyronald** [comment]
@rbardini 	You are right , I actually forgot to add that to the options page.

I released v3.15.1 just now with a fix, check it out
